## Statistical analysis for crime data

##### Import libralies and load dataset

In [1]:
# import necessary libraries
import pandas as pd
import numpy as np

In [2]:
df = pd.read_csv('../dataset/cleaned_data.csv')
df.head()

Date                  Measure               Borough  \
0  2023-06-01  Business Crime Outcomes  Barking and Dagenham   
1  2023-06-01  Business Crime Outcomes  Barking and Dagenham   
2  2023-06-01  Business Crime Outcomes  Barking and Dagenham   
3  2023-06-01  Business Crime Outcomes  Barking and Dagenham   
4  2023-06-01  Business Crime Outcomes                Barnet   

                 Crime Section            Crime group  \
0        POSSESSION OF WEAPONS  POSSESSION OF WEAPONS   
1                        THEFT            OTHER THEFT   
2                        THEFT            SHOPLIFTING   
3  VIOLENCE AGAINST THE PERSON   VIOLENCE WITH INJURY   
4    ARSON AND CRIMINAL DAMAGE        CRIMINAL DAMAGE   

                                             Outcome Positive Outcome  \
0  Investigation Complete; No Suspect Identified....                N   
1  Investigation Complete; No Suspect Identified....                N   
2  Named Suspect Identified: Victim Supports Poli...                N   
3  Named Suspect Identified: Evidential Difficult...                N   
4  Named Suspect Identified: Victim Supports Poli...                N   

   Outcome Count Refresh Date  Year  Month   Weekday  
0              1   2025-06-05  2023      6  Thursday  
1              2   2025-06-05  2023      6  Thursday  
2              2   2025-06-05  2023      6  Thursday  
3              2   2025-06-05  2023      6  Thursday  
4              1   2025-06-05  2023      6  Thursday

#### Mean crime count per borough

In [ ]:
df['Borough'].value_counts().mean()

np.float64(12347.5)

The mean is the sum of all values divided by the number of values and it is the average. The average number of crime in borough is 12347

#### Median crime count per borough

In [5]:
df['Crime Section'].value_counts().median()

np.float64(23347.0)

The Median is the middle value in a dataset. 

#### Standard Deviation

In [6]:
df['Outcome Count'].std()

np.float64(3.3720163769819123)

Standard deviation measures how spread out the data is. A high standard deviation means more variability.
This data has standard deviation of 3.37 which is moderately spread out. 